android/assets/jsons/Civ V - Vanilla/Nations.json

```diff
@@ -340,10 +340,10 @@
 		"uniques": ["67% chance to earn 25 Gold and recruit a Barbarian unit from a conquered encampment", "-[25]% [Land] unit maintenance costs"],
 		"cities": ["Berlin","Hamburg","Munich","Cologne","Frankfurt","Essen","Dortmund","Stuttgart","Dusseldorf","Bremen",
 			"Hannover","Duisburg","Leipzig","Dresden","Bonn","Bochum","Bielefeld","Karlsruhe","Gelsenkirchen","Wiesbaden",
-			"Munster","Rostok","Chemnitz","Braunschweig","Halle","Mצnchengladbach","Kiel","Wuppertal","Freiburg","Hagen",
+			"Münster","Rostock","Chemnitz","Braunschweig","Halle","Mönchengladbach","Kiel","Wuppertal","Freiburg","Hagen",
```

Comment:
As far as I know you should use the english names/translations here. Right?
These name will then be written by the TranslationWriter to the langugage files and everyone can translate it to their languages.

Reply:
Precisely. So "Mönchenglatzbach = " should appear automatically in English.properties instead of what I removed - a while later - and promptly be ignored. When the key is already there - like in most of the files changed herein - I hope the TranslationWriter will simply let it be.

Reply:
I'm confused 🤯
I thought that only english words will be in the JSON files and the TranslationWriter will copy them as "to translate" strings into the translation files.
So from my point of view, "Munster" should be in the Nations.json and after TRWriter did its job, we can change "Munster = " to "Munster = Münster"

Reply:
Only if english natives are already used to the "Munster" form somehow - at least that was my reasoning. When 99% of them never heard of Mülheim they should get Mülheim... So Munich was clearly the prejudiced category, Dusseldoof borderline, and Munster - no the horror movie doesn't count?

Reply:
Ok. Do it like that.